ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -28,7 +28,7 @@
           "machines": {
             "dev-machine": {
               "installers": [
-                "org.eclipse.che.exec", "org.eclipse.che.terminal", "org.eclipse.che.ws-agent"
+                "org.eclipse.che.exec", "org.eclipse.che.terminal", "org.eclipse.che.ws-agent", "org.eclipse.che.ls.java"
```

Comment:
This turns on the java ls per default in those workpaces, correct?

Reply:
Right.